### DISCUSSION TITLE
how to disable base64 in ckeditor??

### DISCUSSION THREAD

**Discussion by leo719044** [YetiForceCompany/YetiForceCRM#15208] [How to]
I would like to save image directory as image instead of base64 in database, because multiple images are required in ticket, and it definately makes database very big if images are stored in base64

Anyone could tell me how to do it??

Thanks in advance.

**Comment by mariuszkrzaczkowski** [comment]
remove "ckeditor-image-to-base" from 
https://github.com/YetiForceCompany/YetiForceCRM/blob/7790ee4508ca589116b1f2f1f067a30b0d7a4cb3/public_html/layouts/resources/Fields.js#L533-L573

**Reply by leo719044** [reply]
Thanks for your help. 

Unfortunately, I modified the files you suggested. But I still get the old file after clearing cache both in browser and server folder.

Could you please kindly suggest how to update the javsscript file?

**Reply by mariuszkrzaczkowski** [reply]
Check that you have disabled https://doc.yetiforce.com/code/classes/Config-Developer.html#property_MINIMIZE_JS

**Reply by leo719044** [reply]
thank you so much for your support.

**Comment by vovpff** [comment]
@mariuszkrzaczkowski but ho to automatic upload images in this case?

**Reply by mariuszkrzaczkowski** [reply]
there is no ready functionality, it would have to be added

**Reply by leo719044** [reply]
now I can upload the image successfully to server. but after saving , hyperlink like "scr" or "href" got deleted. 

I supposed it might be blocked b the settings of ceditors.

**Reply by rskrzypczak** [reply]
This are CSP restrictions.
Links[href]: https://doc.yetiforce.com/code/classes/Config-Security.html#property_purifierAllowedDomains
Images: https://doc.yetiforce.com/code/classes/Config-Security.html#property_allowedImageDomains

**Reply by leo719044** [reply]
thank you so much

**Reply by leo719044** [reply]
@rskrzypczak thanks for your reply, I tried, but unfortunately it does not work.

I made it with the path in image url starting /upload/imgexxxx.png insteal of full link
